packages/@react-spectrum/color/stories/ColorField.stories.tsx

```diff
@@ -21,100 +21,114 @@ import {useId} from '@react-aria/utils';
 import {View} from '@react-spectrum/view';
 import {VisuallyHidden} from '@react-aria/visually-hidden';
 
+const parameters = {
+  args: {
+    allowsAlpha: false
+  },
+  argTypes: {
+    allowsAlpha: {
+      control: {type: 'boolean'}
+    }
+  }
+};
+
 storiesOf('ColorField', module)
+  .addParameters(parameters)
   .add(
     'Default',
-    () => render()
+    (args) => render(args)
   )
   .add(
     'has default value',
-    () => render({defaultValue: '#abcdef'})
+    (args) => render({...args, defaultValue: '#abcdef'})
   )
   .add(
     'value',
-    () => render({
+    (args) => render({
+      ...args,
       value: '#FF00AA',
       onChange: action('change')
     })
   )
   .add(
     'isQuiet',
-    () => render({isQuiet: true})
+    (args) => render({...args, isQuiet: true})
   )
   .add(
     'isReadOnly',
-    () => render({isReadOnly: true, defaultValue: '#abcdef'})
+    (args) => render({...args, isReadOnly: true, defaultValue: '#abcdef'})
   )
   .add(
     'isDisabled',
-    () => render({isDisabled: true, defaultValue: '#abcdef'})
+    (args) => render({...args, isDisabled: true, defaultValue: '#abcdef'})
   )
   .add(
     'validationState valid',
-    () => render({validationState: 'valid'})
+    (args) => render({...args, validationState: 'valid'})
   )
   .add(
     'validationState invalid',
-    () => render({validationState: 'invalid'})
+    (args) => render({...args, validationState: 'invalid'})
   )
   .add(
     'required, label, optional',
-    () => (
+    (args) => (
       <Flex direction="column" gap="size-100">
-        {render({isRequired: 'true'})}
-        {render({isRequired: 'true', necessityIndicator: 'label'})}
-        {render({necessityIndicator: 'label'})}
+        {render({...args, isRequired: 'true'})}
+        {render({...args, isRequired: 'true', necessityIndicator: 'label'})}
+        {render({...args, necessityIndicator: 'label'})}
       </Flex>
     )
   )
   .add(
     'controlled value',
```

Comment:
This story is blowing up: 
<img width="660" alt="image" src="https://user-images.githubusercontent.com/9661127/191329729-7ea57dd1-6bee-417e-b0f1-1e2178733547.png">

Reply:
Dogs allows

Reply:
ah, thanks, i'll add a test for it